src/editorview.h

```diff
@@ -1,7 +1,10 @@
 #ifndef DL_EDITORVIEW_H
 #define DL_EDITORVIEW_H
 
+#include "qevent.h"
```

Comment:
This should be `#include <QMouseEvent>`, and it should be in the .cpp file. (You should use as few #include statements in .h files as possible. It speeds up compilation.)